ledger-api/grpc-definitions/com/daml/ledger/api/v1/transaction.proto

```diff
@@ -72,7 +72,7 @@ message TransactionTree {
 message TreeEvent {
     oneof kind {
         CreatedEvent created = 1;
-        ExercisedEvent exercised = 2;
+        ExercisedEvent exercised = 2; // should be introduced into transactions without children; needs modification
```

Comment:
@mzagorski-da : what kind of modification are you thinking about? Can you change the PR to contain it, so we can align on it?